docs/strategy-callbacks.md

```diff
@@ -611,7 +611,7 @@ class DigDeeperStrategy(IStrategy):
     # ... populate_* methods
```

Comment:
we'll want to have some explanation in this section (above this codeblock, obviously) - explaining that the callback won't be called when `max_buy_position_adjustment` is set and reached for a trade.